var/spack/repos/builtin/packages/esmf/package.py

```diff
@@ -88,6 +88,7 @@ class Esmf(MakefilePackage):
         when="@8.3.0b09",
     )
     variant("debug", default=False, description="Make a debuggable version of the library")
+    variant("caplinks", default=False, description="Create symlinks libESMF.{a,so}")
```

Comment:
I don't think we should do this. We need to figure out how the `-lESMF` comes in, and fix it.